airbyte-api-server/src/main/java/io/airbyte/api/server/forwardingClient/InternalApiWrapper.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.api.server.forwardingClient;
+
+import io.airbyte.api.client.model.generated.ConnectionIdRequestBody;
+import io.micronaut.context.annotation.Bean;
+import io.micronaut.http.HttpResponse;
+import java.util.UUID;
+import javax.validation.constraints.NotBlank;
+import lombok.extern.slf4j.Slf4j;
+
+@Bean
```

Comment:
FYI, we have been standardizing on the Jakarta `Singleton` annotation.  Micronaut honors both.